ci/playbooks/kuttl/deploy-deps.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Run ci_framework bootstrap playbook
+  ansible.builtin.import_playbook: "../../../playbooks/01-bootstrap.yml"
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  name: install dev tools
+  tasks:
+    - name: Download install_yamls deps
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_download_tools'
+
+- name: Run ci_framework infra playbook
+  ansible.builtin.import_playbook: "../../../playbooks/02-infra.yml"
+
+- name: Build dataset hook
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  connection: local
+  tasks:
+    - name: Assert that operator_name is set
+      ansible.builtin.assert:
+        that:
+          - operator_name is defined
```

Comment:
If we move this to line 8 we can safe some time/resources/bandwith if the assertion is not met.

Reply:
done, thanks